### PR TITLE
updated the run classification

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/EPR.Calculator.API.UnitTests/BaseControllerTest.cs [hunk 1]
```diff
@@ -687,7 +688,7 @@ protected static IEnumerable<CalculatorRun> GetCalculatorRuns()
             list.Add(new CalculatorRun
             {
                 Id = 2,
-                CalculatorRunClassificationId = 2,
+                CalculatorRunClassificationId = (int)RunClassification.Running,
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API.UnitTests/BaseControllerTest.cs:691]: Can you make it consistent, either by adding a space in between or removing the space in between :) ?

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs [hunk 2]
```diff
@@ -96,7 +97,7 @@ public async Task<IActionResult> Create([FromBody] CreateCalculatorRunDto reques
                         Financial_Year = request.FinancialYear,
                         CreatedBy = request.CreatedBy,
                         CreatedAt = DateTime.Now,
-                        CalculatorRunClassificationId = 1,
+                        CalculatorRunClassificationId = (int)RunClassification.Running,
```
Comment (johnbenjamin-defra) [inline src/EPR.Calculator.API/Controllers/CalculatorController.cs:100]: Might change based on what you decide for my above comment.

File: src/EPR.Calculator.API/Enums/RunClassification.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿namespace EPR.Calculator.API.Enums
```
Comment (udaydefra) [inline src/EPR.Calculator.API/Enums/RunClassification.cs:1]: Please replace this 
`    public enum RunClassiciations
    {
        [Description("IN THE QUEUE")]
        INTHEQUEUE = 1,
        [Description("RUNNING")]
        RUNNING,
        [Description("UNCLASSIFIED")]
        UNCLASSIFIED,
        [Description("PLAY")]
        PLAY,
        [Description("ERROR")]
        ERROR
    }`

**Review by johnbenjamin-defra — CHANGES_REQUESTED** [review]

File: src/EPR.Calculator.API/Enums/RunClassification.cs (hunk 3 above)
Comment (Skumarhpa) [reply src/EPR.Calculator.API/Enums/RunClassification.cs]: resolved the comment.

File: src/EPR.Calculator.API.UnitTests/BaseControllerTest.cs (hunk 1 above)
Comment (Skumarhpa) [reply src/EPR.Calculator.API.UnitTests/BaseControllerTest.cs]: made it consistent

File: src/EPR.Calculator.API/Controllers/CalculatorController.cs (hunk 2 above)
Comment (Skumarhpa) [reply src/EPR.Calculator.API/Controllers/CalculatorController.cs]: made it consistent

**Review by udaydefra — APPROVED** [review]
LGTM

**Review by johnbenjamin-defra — APPROVED** [review]

**Pull request merged**